src/node-lib/curriculum-api-2023/queries/unitListing/convertSubjectToSlug.ts

```diff
@@ -0,0 +1,14 @@
+export function convertSubjectToSlug(subject: string): string {
```

Comment:
Had this refactored to be a map of Subjects and corresponding slugs, however, function could return undefined which was throwing so strange type errors further down the line.